R/manual_layer_execute.R

```diff
@@ -37,18 +38,19 @@ execute_generic_udf <- function(namespace, udf, args=NULL) {
   parsed[["value"]] # Extract the value field
 }
 
-##' TileDB Cloud UDF-Execution Helper for Single-Array UDFs
+##' Execute a single-array UDF on TileDB Cloud
+##'
+##' This invokes a user-defined function in TileDB Cloud.
 ##'
-##' This function invokes a user-defined function in TileDB Cloud.
-##' Nominally you will first call login(); if not, the results
-##' of the last login at ~/.tiledb/cloud.json will be used.
+##' Nominally you will first call \code{\link{login}}; if not, the results
+##' of the last login at \code{~/.tiledb/cloud.json} will be used.
 ##'
 ##' All arguments are required.
 ##'
 ##' @param namespace Namespace within TileDB cloud.
 ##'
 ##' @param array Name of the array. E.g. if the URI is \code{tiledb://hello/world}
-##' then the namespace is "hello" and the array is "world".
+##' then the namespace is \code{hello} and the array is \code{world}.
```

Comment:
I write docs this way, the "kids these days" enable markdown with roxygen for Rd.  Just thought I'd mention that.